src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/ConstMetadata.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Metadata\SchemaMetadata;
+
+class ConstMetadata extends PropertyMetadata
+{
+    /**
+     * @var string|number|null
```

Comment:
number?

Reply:
What do you want to say by that? A field in a form can return about anything, so we should at least cover these two (I don't imagine right now that people are using this in combination with objects as well).

Reply:
Number does not exist as type in PHP. https://docs.phpdoc.org/guides/types.html
PHP only know about int|float